Adds 3 parameter scaling function to MatrixTransformStack

MehVahdJukaar opened the pull request:
Very simple PR to allow one to scale a MatrixTransformStack on multiple axis

MehVahdJukaar committed:
Added 3 parameter scaling function

Simply adds a 3 parameter function to to MatrixTransformStack so a matrix can be scaled on different axis

MehVahdJukaar committed:
Merge pull request #1 from MehVahdJukaar/MehVahdJukaar-patch-1

Add 3 parameter MatrixTransformStack scaling function

Jozufozu commented on src/main/java/com/jozufozu/flywheel/util/transform/MatrixTransformStack.java:
```diff
@@ -41,6 +41,12 @@ public TransformStack scale(float factor) {
 		return this;
 	}
+	
+	@Override
```
Doesn't seem like this is overriding anything, but it probably should be.
Let's change the current `TransformStack#scale` to accept xyz args, and have the single argument version be a default implementation in `TransformStack`. Then `MatrixTransformStack` only needs to override the xyz arg version.

MehVahdJukaar committed:
Update MatrixTransformStack.java

MehVahdJukaar committed:
Update TransformStack.java

Jozufozu committed:
Restore single paramater scale function

 - Now has default impl in TransformStack
 - MatrixTransformStack now only overrides xyz scale
